src/bot/Bot.js

```diff
@@ -64,6 +69,10 @@ export default class Bot {
     return this._handler;
   }
 
+  get mapPageToAccessToken(): ?(pageId: string) => Promise<string> {
```

Comment:
We should pass mapPageToAccessToken into connector, not here. 
And should not export getter like this, because bot will never know what is a page and so does LineBot, SlackBot and TelegramBot.

Reply:
Got it. I will make some changes.